### DISCUSSION TITLE
i3 keybindings not working while Rofi is open

### DISCUSSION THREAD

**Discussion by redevined** [i3/i3#5175] [Q&A] [answered]
Hi,
not sure if this is a Rofi or i3 issue, but the keybindings defined in i3 don't work while Rofi is open. E.g.:

i3 config
`bindsym $mod+Return exec i3-sensible-terminal`
Rofi command
`rofi -show run`
=> Then the keybinding `$mod+Return` doesn't have any effect.

This is also true for keybinding calling external scripts or changing volume etc.
Is there a fix for this? Thanks in advance!

**Comment by Airblader** [comment, marked as answer]
That's working as intended, and no, there's nothing you can do other than close rofi.

**Reply by redevined** [reply]
Ok thanks 👍

**Reply by dxrcy** [reply]
Can I ask why this is intended? Is it a limitation or a decision?

**Reply by orestisfl** [reply]
A limitation, rofi grabs the keyboard and i3 cannot receive bindings.

**Reply by dxrcy** [reply]
Thanks!
